module2/projects/hang-in-there-api/evaluation.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: Hang in There API Evaluations
+length: 1 week
+type: project
+---
+_[Back to Hang in There API Home](./index)_
+
+## Presentation
+For the project evaluation, each project pair should prepare presentation. Your presentation should cover the following points:
+
+* Demonstration of functionality via Postman suites
+* Technical quality and organization of the code, identifying code that should be refactored and how it would be refactored
+* Running your application's test suite and a discussion of test coverage
+* Identifying the area(s) of code of which you are most proud, and an area where you would like specific feedback
+
+_All_ team members are expected to participate equally in the presentation. Students should focus on practicing technical communication that is succinct and utilizes appropriate technical vocabulary.
+
+Slides are not required, but encouraged as a way to facilitate the presentation along with sharing specific code examples.
+
+
+## Rubric
+Your project will be evaluated based on the following rubric: 
+
+### Feature Delivery
+
+* **Exceeds Expectations**: Project completes all requirements and extensions.
+* **Meets Expectations**: Project completes all requirements
+* **Approaching Expectations**: Project fails to complete 1 - 2 required endpoints
+* **Below Expectations**: Project fails to complete more than 3 or more endpoints
+
+### Test Driven Development
+
+* **Exceeds Expectations**: Project achieves 100% test coverage at the unit and integration levels. A gem that enhances testing effectiveness is implemented (orderly, factorybot, faker, etc) throughout tests.
+* **Meets Expectations**: Project achieves greater than 90% test coverage
+* **Approaching Expectations**: Project achieves greater than 80% test coverage
+* **Below Expectations**: Project does not have 80% test coverage.
+
+### Rails
+
+* **Exceeds Expectations**: Project demonstrates exceptionally well factored code. Students implement strategies not discussed in class and can defend their design decisions (callbacks, scopes, etc)
+* **Meets Expectations**: Students use the principles of MVC to effectively organize code with only 1 - 2 infractions. Routes and Actions mostly follow RESTful conventions. Serializers are used for formatting JSON responses.
```

Comment:
Is this Meets Expectations bullet still worded in a way that is applicable to just the api? Not a monolith?  (I genuinely don't know but thought api only took us slightly away from strict MVC architecture)

Reply:
Yes! I combined a few of the old mod 2 rubrics and then adjusted for an API only app.